api/Element.json

```diff
@@ -6627,7 +6630,8 @@
             ],
             "opera_android": [
               {
-                "version_added": "24"
+                "version_added": "24",
+                "notes": "Version 65 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Comment:
```suggestion
                "notes": "From version 65 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```